scipy/spatial/transform/_rotation.pyx

```diff
@@ -3483,13 +3483,32 @@ cdef class Rotation:
             # We first find the minimum angle rotation between the primary
             # vectors.
             cross = np.cross(b_pri[0], a_pri[0])
-            theta = atan2(_norm3(cross), np.dot(a_pri[0], b_pri[0]))
-            if theta < 1e-3:  # small angle Taylor series approximation
+            cross_norm = _norm3(cross)
+            theta = atan2(cross_norm, _dot3(a_pri[0], b_pri[0]))
+            tolerance = 1e-3  # tolerance for small angle approximation (rad)
+            R_flip = cls.identity()
+            if abs(np.pi - theta) < tolerance:
```

Comment:
Should we remove `abs` here? I think `atan2 <= pi`  is guaranteed when the first argument is positive.

With `abs` the following logic is not very clear.

Reply:
I agree, I'm just not sure if the np.pi and libc.math atan2 limit of +pi are exactly the same floating point number. The absolute value here is trying to protect for if theta is infinitesimally larger than pi. I believe the code here should still work as expected if that's the case.

Reply:
I suppose a negative value here is actually covered by the inequality, so I removed the `abs`.